src/app/server/CommissioningWindowManager.h

```diff
@@ -56,11 +56,22 @@ class CommissioningWindowManager : public SessionEstablishmentDelegate, public a
         return System::Clock::Seconds16(15 * 60);
     }
 
+#if defined(CONFIG_IM_BUILD_FOR_UNIT_TEST) || defined(CHIP_CONFIG_TEST)
+    // For tests only, allow overriding the spec-defined minimum value of the
+    // commissioning window timeout.
+    static Optional<System::Clock::Seconds16> mMinCommissioningTimeoutOverride;
+    static System::Clock::Seconds16 MinCommissioningTimeout()
```

Comment:
We don't want the member to be public.  And we don't need either of these to be static.  The constexpr function needs to be static to actually be constexpr, but this function doesn't have that constraint.